setup.py

```diff
@@ -29,7 +29,7 @@
     # with the LICENSE file added to its root folder
     # and the following patches applied
     # https://github.com/virtuald/pyhcl/pull/57
-    'pyhcl~=0.3',
+    'pyhcl<0.3.14',
```

Comment:
I think a rebase should make dep change go away.

Reply:
Rebased